PowerPointLabs/PowerPointLabs/TimerLab/TimerLabPaneWPF.xaml.cs

```diff
@@ -97,11 +113,21 @@ private int LineMarkerColor()
         #endregion
 
         #region Timer Creation
-        private void CreateBlocksTimer(int duration, float timerWidth, float timerHeight, float timerLeft, float timerTop, bool isCountdown)
+        private void CreateBlocksTimer(int duration, float timerWidth, float timerHeight, float timerLeft,
+            float timerTop, bool isCountdown, bool isProgressBar)
         {
-            AddTimerBody(timerWidth, timerHeight, timerLeft, timerTop, TimerBodyColor());
+            if (isProgressBar)
+            {
+                AddTimerBody(timerWidth, timerHeight, timerLeft, timerTop, SliderColor());
+                AddProgressBar(duration, timerWidth, timerHeight, TimerBodyColor(), SlideWidth());
+            }
+            else
+            {
+                AddTimerBody(timerWidth, timerHeight, timerLeft, timerTop, TimerBodyColor());
+                AddSlider(duration, timerWidth, timerHeight, ProgressBarTimerBodyColor(), SlideWidth());
```

Comment:
Did you mix up `ProgressBarTimerBodyColor()` and `TimerBodyColor()` here? Seems like you are using `ProgressBarTimerBodyColor()` when `isProgressBar` returns false

Reply:
The color returns correctly but the naming is wrong. Will change it!